### PR TITLE
Fix: auto update not works when using Pipeline

### LINKED ISSUE #89: When coverage goes down threshold is updated and the build is not marked as unstable
Using the snippet generator I've added the following step to my `Jenkinsfile`:

```
cobertura coberturaReportFile: 'coverage.xml', conditionalCoverageTargets: '80, 0, 0', failUnhealthy: true, failUnstable: true, lineCoverageTargets: '80, 0, 0', maxNumberOfBuilds: 0, methodCoverageTargets: '80, 0, 0', onlyStable: false, sourceEncoding: 'ASCII', zoomCoverageChart: false
```

This should set the initial threshold to 80% for lines, methods and conditionals and auto-update the minimum when the build is successful (as `autoUpdateHealth` and `autoUpdateStability` are `true` by default). But when coverage goes down the threshold is updated and the build is not marked as unstable:

The pipeline steps reports the update of the thresholds:
```
    Methods's new health minimum is: 78.84
    Lines's new health minimum is: 82.19
    Conditionals's new health minimum is: 72.23
    Methods's new stability minimum is: 78.84
    Lines's new stability minimum is: 82.19
    Conditionals's new stability minimum is: 72.23
```

In other non-pipeline jobs using the `cobertura-plugin`, when the coverage is bellow the minimum the coverage does not get updated and the build is marked as unstable.

**Environment**

- Jenkins v.2.73.3
- Cobertura plugin v1.12
- Pipeline plugin v2.5
- Pipeline: API plugin v2.23.1

### PR DESCRIPTION
**Pull request by cizezsy**
Fix #89 and #72.

### CODE REVIEW

**Comment by lexotero** [comment]
Is there anything I can do to help moving this forward?

**Comment by jeffpearce** [comment]
@lexotero you did it :-) Sorry, I have so much on my plate I dropped this, but seeing your comment in my inbox reminded me

File: pom.xml [hunk 1]
```diff
@@ -29,9 +29,9 @@
   </licenses>
 
   <scm>
-   <connection>scm:git:git://github.com/jenkinsci/cobertura-plugin.git</connection>
-   <developerConnection>scm:git:git@github.com:jenkinsci/cobertura-plugin.git</developerConnection>
-   <url>https://github.com/jenkinsci/cobertura-plugin</url>
+    <connection>scm:git:git://github.com/jenkinsci/cobertura-plugin.git</connection>
+    <developerConnection>scm:git:git@github.com:jenkinsci/cobertura-plugin.git</developerConnection>
+    <url>https://github.com/jenkinsci/cobertura-plugin</url>
```
Comment (jeffpearce) [inline pom.xml]: Could you revert this change, since it's only whitespace?

Comment (jeffpearce) [reply pom.xml]: Actually, never mind. No need to delay the PR for that

**Comment by jeffpearce** [comment]
@cizezsy can you do a release?

**Comment by cizezsy** [comment]
@jeffpearce OK, I have released a new version :P thanks~

**Comment by lexotero** [comment]
Many thanks guys!!! This will make our life way easier.
